src/foam/lib/json/UnknownFObjectParser.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package foam.lib.json;
+
+import foam.lib.parse.*;
+import foam.core.X;
+
+public class UnknownFObjectParser implements Parser {
```

Comment:
This doesn't work for all valid JSON. Ex.:
{key:"}"}
Open ticket to fix and assign to yourself. I'll explain how to fix.